client/src/modules/policy/pages/policyPage.tsx

```diff
@@ -1,15 +1,44 @@
 import React from 'react';
 import { NextPage } from 'next';
 import { Flex, Heading, Text } from '@chakra-ui/react';
+import { LinkButton } from 'chakra-next-link';
 
 export const PolicyPage: NextPage = () => {
+  const serverUrl =
+    process.env.NEXT_PUBLIC_SERVER_URL || 'http://localhost:5000';
+
```

Comment:
I am getting hydration error, asking me to check.  https://nextjs.org/docs/messages/react-hydration-error

Does adding these lines change how the type of the page? That's why I am getting hydration error?

And do I add `useEffect`, know how `useEffect` works, it feels overkill for something that should be update once 🤔

Reply:
I don't know why exactly hydration error gives troubles here. But after putting the _Authenticate with Google_ behind waiting for loading user and checking permission for authentication with google, the issue doesn't appear anymore.